src/app/src/main/kotlin/org/vaccineimpact/orderlyweb/controllers/web/ReportController.kt

```diff
@@ -69,6 +70,20 @@ class ReportController(context: ActionContext,
         return RunReportViewModel(context, metadata, gitBranches)
     }
 
+    fun getListReports(): List<Map<String, Any?>>
+    {
+        val response = orderlyServerAPI.get("/reports/source", context)
```

Comment:
Is no error handling required for this request?

Reply:
I'm not sure there's an error situation that we can recover from - but errors do get propagated up to the UI, consistent with other calls. Keen to learn anything we can learn from HINT etc though, so any ideas welcome!